### ISSUE TITLE
Init links to an HTTPS version, even if Certbot was unsuccessful

### ISSUE THREAD

**Issue by colmtroy** [kovshenin/sail#6]
Howdy!
Was excited to give Sail a try but hit an immediate error with my first try.
Running sail 0.9.12 on macOS 11.2.2 (m1 macbook air)

sail init ran successfully but when I open the WP url https://b0dd8f888b61a6a6.sailed.io/ it's clear nginx isn't running correctly.

Any ideas as to what the issue might be?

**Comment by kovshenin** [comment]
Hi @colmtroy, thanks so much for giving Sail a try!

I'm looking at the provision logs for b0dd8f888b61a6a6 on the API server and it looks like Let's Encrypt failed to issue an SSL certificate for the domain, the service was down for maintenance. And indeed, the site works without HTTPS right now, so Nginx is running for sure. Here's what you could do:

1. You can try and re-issue an SSL certificate for the internal domain:

```
sail domain make-https b0dd8f888b61a6a6.sailed.io
```

2. OR If you haven't put anything worthy on the droplet yet, you can try to destroy and re-provision it:

```
sail destroy
sail init --force
```

I'll keep this issue open for now and probably add an extra check on the API server and some verbosity, for when SSL fails, to make sure users aren't trying to access the site via HTTPS if it hasn't received a valid certificate yet.

Let me know if this doesn't work for you and thanks so much for reporting this!

**Comment by colmtroy** [comment]
@kovshenin `make-https` did the job - thanks for the quick reply!

**Comment by kovshenin** [comment]
Glad it worked, let me know if you run into any other trouble!

**Comment by kovshenin** [comment]
Should be fixed in #14
